packages/communication-react/review/stable/communication-react.api.md

```diff
@@ -4511,6 +4659,8 @@ export type VideoGalleryParticipant = {
     videoStream?: VideoGalleryStream;
     isScreenSharingOn?: boolean;
     spotlight?: Spotlight;
+    mediaAccess?: MediaAccess;
+    role?: string;
```

Comment:
I remember James saying we should keep UI component types agnostic of state's like role. But I think we need it so that the VideoGallery knows which remote participants can have their audio or video be forbidden. Just thinking out loud here.

Reply:
Can we change the role property to be:
```
canAudioBeDisabled?: boolean
canVideoBeDisabled?: boolean
```
The reason for this is Contoso using components only may not know what role is which is an ACS definition.

Reply:
We already have this role in some places, need to display Role on the UI so that user know who the attendees, organizer or presenters are and what actions user can do to other participants.
I am not sure if we want to hide the role from UI on VideoGalleryParticipant.tsx.

![image](https://github.com/user-attachments/assets/58a18a00-cacf-4438-b483-8b2f02565372)
![image](https://github.com/user-attachments/assets/c1764414-f628-4a2e-94c2-f6c5dc76b519)

Reply:
yeah I am in agreeance here with you @mgamis-msft. it is important that we don't place any API typings in the components package that is related to ACS. 

@fuyan2024 one of the important principles of the react-components package is that we don't include any types that are connected to ACS, while we can provide a proxy type here to not depend on the Calling SDK the term Role is specific to ACS, Teams and Rooms and if Contoso is using the component for their own Calling service they will need to implement this concept to have it work. By following @mgamis-msft's suggestion of having state reflecting the permissions we side step this issue as Contoso can use these to reflect media access state.

Reply:
Had meeting to discuss this and team decided to go with canAudioBeForbidden and canVideoBeForbidden.